### PR TITLE
Issue 126 - monitor user location changes

### PR DESCRIPTION
**Pull request by fluffyemily**
Enable background location monitoring and setup significant location change monitoring.
Update places when a location change occurs, being careful not to update while a user is currently browsing places in either Place Details or Place Carousel

#126

### CODE REVIEW

File: Prox/Prox.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -493,8 +493,13 @@
 				TargetAttributes = {
 					7B29E7BE1DA3EC360099AF38 = {
 						CreatedOnToolsVersion = 8.0;
-						DevelopmentTeam = 43AQ936H96;
```
Comment (mcomella) [inline Prox/Prox.xcodeproj/project.pbxproj]: The development team changed.

File: Prox/Prox/Info.plist [hunk 2]
```diff
@@ -18,8 +18,22 @@
 	<string>1.0</string>
 	<key>CFBundleVersion</key>
 	<string>1</string>
+	<key>LSApplicationQueriesSchemes</key>
```
Comment (mcomella) [inline Prox/Prox/Info.plist]: nit: probably automated change but this didn't need to get re-ordered.

File: Prox/Prox/PlaceCarousel/PlaceCarouselViewController.swift [hunk 3]
```diff
@@ -196,9 +199,26 @@ class PlaceCarouselViewController: UIViewController {
...
+
+        guard let location = getCurrentLocation() else {
+            return
+        }
+
+        // this will update places when we first load, and whenever we show the place carousel
+        // but not update places while we are showing details, until we close the place details
+        // this ensures consistency between the place details and the carousel underneath
+        // and makes sure we don't end up providing weird data to users while they are scrolling
+        // through places details
```
Comment (mcomella) [inline Prox/Prox/PlaceCarousel/PlaceCarouselViewController.swift]: Cool – good plan!

File: Prox/Prox/Info.plist (hunk 2 above)
Comment (fluffyemily) [inline Prox/Prox/Info.plist]: yeah, automated change.

**Comment by fluffyemily** [comment]
[As a user I want the app to be aware when my location changes even when I am not using the app so that I can be informed when new places and events are near me as I move about](https://trello.com/c/NESzoTGn/139-as-a-user-i-want-the-app-to-be-aware-when-my-location-changes-even-when-i-am-not-using-the-app-so-that-i-can-be-informed-when-ne)

**Pull request merged**